docs/docsite/rst/windows_winrm.rst

```diff
@@ -657,6 +656,29 @@ for each authentication option. See the section on authentication above for more
     encryption done over TLS. The WinRM payload is still encrypted with TLS
     when run over HTTPS, even if ``ansible_winrm_message_encryption=never``.
 
+IPv6 Addresses
+``````````````
+IPv6 addresses can be used instead of IPv4 addresses or hostnames and is set
+normally in an inventory. Ansible will attempt to parse the address using the
+`ipaddress <https://docs.python.org/3/library/ipaddress.html>`_ package and
+pass to pywinrm correctly.
+
+When defining a host using an IPv6 address, just put the address like you would
```

Comment:
suggest changing to "just add the IPv6 address as you would an IPv4 address or hostname::"